CMakeLists.txt

```diff
@@ -848,6 +848,13 @@ endif()
 
 # Add all of the subdirectories, where we actually do work.
 
+# Include stdlib/ here because any targets defining IS_STDLIB
+# or TARGET_LIBRARY are added as implicit dependencies of the
+# swift-stdlib-* set of targets, but only after those targets have been
```

Comment:
Can you explain in the comment why this is connected to tools. There is some sort of implicitness in your reasoning.